bundle/Resources/config/default_settings.yml

```diff
@@ -5,7 +5,7 @@ parameters:
     ezpublish_legacy.default.module_default_layout: ~
 
     # Pagelayout to use while rendering a content view in legacy
-    ezpublish_legacy.default.view_default_layout: EzPublishLegacyBundle::legacy_view_default_pagelayout.html.twig
+    ezpublish_legacy.default.view_default_layout: "@@EzPublishLegacy/legacy_view_default_pagelayout.html.twig"
```

Comment:
Why a double `@` here?

Reply:
Without it, Symfony crashes with unknown service error or something similar. I can't find the doc about it ATM, but I have it working like this on some of our other projects, after I've read it in the docs 🙊

Reply:
Ha, found it! 

http://symfony.com/doc/current/service_container/parameters.html

> If you use a string that starts with @ or has % anywhere in it, you need to escape it by adding another `@` or `%`

```
# config/services.yaml
parameters:
    # This will be parsed as string '@securepass'
    mailer_password: '@@securepass'

    # Parsed as http://symfony.com/?foo=%s&amp;bar=%d
    url_pattern: 'http://symfony.com/?foo=%%s&amp;bar=%%d'
```